wm/frame.go

```diff
@@ -248,6 +248,11 @@ func (f *frame) maximizeApply() {
 }
 
 func (f *frame) unmaximizeApply() {
+	if f.client.restoreWidth == 0 && f.client.restoreHeight == 0 {
+		screen := f.client.wm.screensProvider.ScreenForWindow(int(f.client.restoreX), int(f.client.restoreY))
```

Comment:
What's the point in passing in the restore parameters - if the width and height are 0 then this is going to be a pretty redundant call... surely the active screen would be more appropriate?

Reply:
Being safe and covering a case where x, y may exist (i.e. top left or corner of a second screen) but w, h are 0